README.md

```diff
@@ -1,189 +1,53 @@
-
-> This is the first html and css project of the first block in which linters are integrated.
-> This project is part of my Microverse exercises, where we learn how to set a up a new repository in a professional manners.
-> This project doesn't require any complex coding. The goal here is to master all of the tools and best practices that are used in a professional development environment.
-
-
-**Portfolio:** is a project where no complex coding is required and the only goal here is to master all of the tools and best practices we learned at Microverse.
-
-## 🛠 Built With <a name="built-with"></a>
-
-### Tech Stack <a name="tech-stack"></a>
-
-<details>
-  <summary>Client</summary>
-  <ul>
-    <li><a href="https://html.com/">index.html</a></li>
-    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">style.html</a></li>
-  </ul>
-</details>
-
-<!-- Features -->
-
-### Key Features <a name="key-features"></a>
-
-- User login form
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-- coming soon
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
-## 💻 Getting Started <a name="getting-started"></a>
-
-
-To get a local copy up and running, follow these steps.
+This is project is the mobile first portifolio project and it has some further testing stages on different types of browsers.
 
 ### Prerequisites
 
-In order to run this project you need:
-
-Installation of Git
-installation of NPM
-a working browser
-live server
-Internet connection 
-A code editor like Visual Studio Code are required.
+Good knowledge of HTML, CSS, Figma and git/Github is required
 
 ### Setup
 
-Clone this repository to your desired folder:
-
-
-```sh
-  cd my-folder(your working machine)
-  git clone git@github.com:Fentahunn/finish_mobile_version.git
-
-```
-
-
-### Install
-
-Install this project with:
-
-
-```sh
-  cd my-project
-  To install Webhint
-  npm init -y
-  npm install --save-dev hint@7.x
-  To install Stylelint
-  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
-
-```
+Any code editor can be used, although in this instance, VS code is used.
 
-### Usage
-
-To run the project, open the project by launching the Live Server plugging in visual studio code.
-
-
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
+## Authors
 
 👤 **Fentahun Agmas**
-👤 **henry cadenaec**
-- GitHub: [@githubFentahun](https://github.com/Fentahunn)
-- Twitter: [@twitterFentahun](https://twitter.com/FentahunAgmas)
-- Github: [@githubhenrycadenaec](https://github.com/henrycadenaec)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+- GitHub: [@githubhandle](https://github.com/Fentahunn)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/fentahun-agmas-5a01aa162/)

```

Comment:
- [ ] Its mandatory to use a microverse readme template for a professional reasons ,Kindly edit your readme to the microverse template and fill all neccesaries sections ,you can follow step by step guild below to fix this .README file should be descriptive:

All main headers from the [README template](https://github.com/microverseinc/readme-template) should be present in a student's repo. (if any relevant header is missing, kindly mark the PR as invalid)

The only optional section is FAQ.
Students should delete the live demo section if the student decides not to deploy the application. If project requirements say students should deploy, they should put the link.
Make sure the student has added the section Future Features, including 1-3 potential features to be added.


Students should remove the section description, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.
It is the student's choice regarding how much into the details they go in each section of the README, you can advise but the student does not have to follow your suggestions.
Students should remove template logo. Optionally, the students can replace with their own logo if they have one.

this is a professional readme screenshot below, pls make yours  to be like this before you resubmit make sure all link are not broken e.g MIT license. 
- ![PREFESSIONAL Readme](https://github.com/Fentahunn/contact_form_final/assets/92458236/18f4bd35-d401-4e9b-be73-5b01d9d10513)